### PR TITLE
Kristall: x86 support, less RAM for building, etc.

### PR DESCRIPTION
**Pull request by win8linux**
This PR updates Kristall to the latest Git revision (as of 22-09-2021) which brings [a fix to large amounts of RAM being used during building and x86 support for Haiku specifically](https://github.com/MasterQ32/kristall/commit/8a7dfe0b1ab9783fddb93d890ea4b856eb93db60). Some other notable changes include:

* [Seperate 'icon theme' preference](https://github.com/MasterQ32/kristall/commit/9bf17318f91b0d5a42c970491d0963178fd23938)
* [Basic root/parent navigation options to menu with shortcuts](https://github.com/MasterQ32/kristall/commit/b82ecad5462eec767e2e3216e9e96dfebf04f1cc) (Root shortcut was later [changed to `Alt+/`](https://github.com/MasterQ32/kristall/commit/fb230927ab64ebcb4f2fe75f33658127421738b9)) along with [toolbar buttons](https://github.com/MasterQ32/kristall/commit/42813aa9ef1b6ada776cf0938ba02dad48a17422) for them
* [Text-search support for Gopher](https://github.com/MasterQ32/kristall/commit/c23988a2ec002042f9db0f8765524703662dcfe6)
* [Emoji toggle preference](https://github.com/MasterQ32/kristall/commit/23feba7e5a1396110d543a676e018bcbdfa50d39)
* [Proper multi-window suppoort](https://github.com/MasterQ32/kristall/commit/9457e720f24d8365ed6a1e82b016078b0bc17eb6)
* [New HTML rendering via HTML code generation](https://github.com/MasterQ32/kristall/commit/4e18c888b159af48bed5ef5f1ebef80e48cd494a)
* [Settings UI cleanup](https://github.com/MasterQ32/kristall/commit/afd25186e03d0d8f35e526acb300a6e43ed6ce41)
* [Simplified image loading](https://github.com/MasterQ32/kristall/commit/c6b7ef61af9f4c7e141c1960dff085e9948d8637)
* [Save settings on exit](https://github.com/MasterQ32/kristall/commit/9a55d32e2c58fbc99e3043fd9ffc29fd62a37db3)
* [Language setting](https://github.com/MasterQ32/kristall/commit/a2f36ec4d14ddf1bcee98e52a0f3a924804d06fb)
* [Basic search engine functionality](https://github.com/MasterQ32/kristall/commit/22c4fb946f5868e48361ebbef6d0dd378087d5ec) and [preference](https://github.com/MasterQ32/kristall/commit/4805c389a7650d85064a3e959936defb8e16968f)
* [Maximum line width fuctionality](https://github.com/MasterQ32/kristall/commit/496ea48de41e6c637c9fb3582216dbd9d8ee55c7)
* [Per-style text width preference](https://github.com/MasterQ32/kristall/commit/19ac43503568ebc7fe4721f84292a11bbf7c2c2c)

### CODE REVIEW

File: www-client/kristall/kristall-0.7.3~git.recipe [hunk 1]
```diff
@@ -6,16 +6,16 @@ Gopher, HTTP/HTTPS, and Finger protocols."
...
 SOURCE_URI="https://github.com/MasterQ32/kristall/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="4f7e3e4d6086a2f6a9e95f8052d3add833046ffb28a2e5371d713a454c1aa134"
+CHECKSUM_SHA256="326d3f3785ba5faf3caf12df1caf99e801816658b79881106e6b39204dfddfc5"
 SOURCE_FILENAME="kristall-$portVersion-tar.gz"
 SOURCE_DIR="kristall-$srcGitRev"
 ADDITIONAL_FILES="kristall.rdef.in"
 
-ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="?x86"
+ARCHITECTURES="!x86_gcc2 x86_64"
```
Comment (Begasus) [inline www-client/kristall/kristall-0.7.3~git.recipe:17]: @waddlesplash we'll be seeing this in the near future ...

Comment (waddlesplash) [reply www-client/kristall/kristall-0.7.3~git.recipe:17]: Indeed, something to watch out for.

Comment (Begasus) [reply www-client/kristall/kristall-0.7.3~git.recipe:17]: We can't all check architectures our selves (don't have a riscV/arm* environment for those), and for those some of these will depend on people with access to them to see if they actually build?

Comment (waddlesplash) [reply www-client/kristall/kristall-0.7.3~git.recipe:17]: The point of `all` is that most software will likely build on riscv or arm if it builds on x86_64. We don't necessarily need to check every single package on every architecture before we publish it; we can just see if it builds and fix it if it needs to be.

**Pull request merged**